### DISCUSSION TITLE
Feature: Debug commas

### DISCUSSION THREAD

**Discussion by zenzeinet** [TabularEditor/TabularEditor3#1096] [Ideas]
Hi,
Regarding the Debug Commas (having commas at the beginning of the line, rather than at the end of the previous line), is that option available? If not, it is coming to TE3? We use it abundantly and the autoformatting would be better with it.
Thanks

**Comment by otykier** [comment]
It should already be available through the right-click menu of any DAX Editor in TE3. We can add a keyboard shortcut too if needed.

**Comment by zenzeinet** [comment]
Hi Daniel,
That's great! I did not look into the right click (was looking at the menus and only saw the long and short options)
Yes, a shortcut or even better a global config option would be great - is that possible?
Thanks
